packages/ckeditor5-dev-release-tools/lib/tasks/commitandtag.js

```diff
@@ -39,6 +41,19 @@ export default async function commitAndTag( { version, files, cwd = process.cwd(
 		return;
 	}
 
+	if ( dryRun ) {
+		try {
+			await git.add( filePathsToAdd );
+			await tools.shExec( 'yarn lint-staged', { cwd, verbosity: 'silent', async: true } );
+		} catch ( e ) {
+			throw e.message;
+		} finally {
+			await git.reset( filePathsToAdd );
+		}
+
+		return;
+	}
```

Comment:
```js
if( dryRun ) {
  // ...
} else {
  // ...
}
```

While both solutions do the same, I would like to keep `if/else` because the part after if applies the opposition behavior to the checked instruction.

Reply:
Adressed in [7859810](https://github.com/ckeditor/ckeditor5-dev/pull/1077/commits/7859810ad6c8a1b9659aa2cc2171824b42507622).